docs/operator-guides/configuring-airbyte.md

```diff
@@ -15,14 +15,12 @@ If you want to manage your own docker files, please refer to Airbyte's docker fi
 
 ## Kubernetes Deployments
 
-The recommended way to run an Airbyte Kubernetes deployment is via the `Kustomize` overlays.
-
-We recommend using the overlays in the `stable` directory as these have preset resource limits.
+The recommended way to run an [Airbyte Kubernetes deployment](../deploying-airbyte/on-kubernetes-via-helm.md) is via the `Helm Charts`.
 
 To configure the default Airbyte Kubernetes deployment, modify the `.env` in the respective directory. Each application will consume the appropriate
```

Comment:
I don't think this is true for Helm charts. As far as I know, the `.env` file has no impact on Helm deployments of Airbyte. Instead, values need to be defined in the `env-configmap.yaml` template.

@git-phu could you sanity check my claim here?

Reply:
Yes that is correct.  Modifying the `.env` file has no impact on Helm deployments.

The instructions for configuring a helm deployment are already present in the linked helm docs here https://docs.airbyte.com/deploying-airbyte/on-kubernetes-via-helm/#custom-deployment

Reply:
@git-phu @pmossman can you check the new sentence? I'll reference to the custom deployment guides too.